benchmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "benchmark",
-  "version": "5.0.2",
+  "version": "5.0.3",
```

Comment:
I am wondering when did we start to change these? Shouldn't they always be 5.0.0? Let's not bump it unless there is a valid reason for it (I don't see one)

Reply:
I think so, for `private` package should be 5.0.0 except framer.